### PR TITLE
Fix bug: Token screen is missing Security Token Address

### PR DESCRIPTION
**Pull request by sajclarke**
Show Security Token Address and etherscan Address link on TokenPage

### CODE REVIEW

File: src/app/token/TokenPage.js [hunk 1]
```diff
@@ -33,15 +33,13 @@ const mapDispatchToProps: DispatchProps = {
...
-type Props = {|
-|} & StateProps & DispatchProps
+type Props = {||} & StateProps & DispatchProps
 
 type State = {|
   isModalOpen: boolean,
 |}
 
 class TokenPage extends Component<Props, State> {
-
```
Comment (bshevchenko) [inline src/app/token/TokenPage.js]: please stop doing this. I told you like months ago that I add this empty line sometimes to make code more readable

File: src/app/token/TokenPage.js [hunk 2]
```diff
@@ -105,32 +102,30 @@ class TokenPage extends Component<Props, State> {
...
                     <br />
                     <CompleteTokenForm onSubmit={this.handleCompleteSubmit} />
                   </div>
                 </div>
-              ) : ''}
-              {token.address ? (
-                <MintTokens />
-              ) : ''}
+              ) : (
+                ''
```
Comment (bshevchenko) [inline src/app/token/TokenPage.js]: and here...
why do you add 2 more lines for this if we only want to show something in one case without else?

File: src/app/token/TokenPage.js [hunk 3]
```diff
@@ -142,9 +137,19 @@ class TokenPage extends Component<Props, State> {
...
                     <p>{token.name}</p>
                   </div>
+                  {token.address ? (
+                    <div className='bx--form-item'>
+                      <label htmlFor='name' className='bx--label'>
+                        Token Address
+                      </label>
+                      <p>{etherscanAddress(token.address)}</p>
+                    </div>
+                  ) : ('')}
```
Comment (bshevchenko) [inline src/app/token/TokenPage.js]: here you have excess ()...

File: src/app/token/TokenPage.js (hunk 2 above)
Comment (sajclarke) [reply src/app/token/TokenPage.js]: I think it's simpler to read since it's harder to miss. Will revert.

File: src/app/token/TokenPage.js (hunk 1 above)
Comment (sajclarke) [reply src/app/token/TokenPage.js]: Fixed.

File: src/app/token/TokenPage.js (hunk 3 above)
Comment (sajclarke) [reply src/app/token/TokenPage.js]: Removed

**Pull request closed without merging**